### DISCUSSION TITLE
New parameter 47260 fan mode for VVM320

### DISCUSSION THREAD

**Discussion by Hoedsch** [yozik04/nibe#68] [Ideas]
Hi,
I've tested 47260 fan mode for the VVM320 sucessfully using the F370 integration.
Would it be possible to enable the parameter for VVM320 too? At the moment it is deactivated for this device in extensions.json.

**Comment by yozik04** [comment]
I see it was added last month.

**Reply by yozik04** [reply]
https://github.com/yozik04/nibe/blob/623695c40a0b6e32530f8ef0399ab52396782bc8/nibe/data/extensions.json#L89

**Reply by Hoedsch** [reply]
Thank you for the information. Then I'm curious when it will appear in the Homeassistant nibe-heatpump integration.

**Reply by yozik04** [reply]
"When" is not a right question for open source project =) I can tell that it is in progress...
